pkgs/build-support/node/fetch-npm-deps/src/main.rs

```diff
@@ -195,6 +298,16 @@ fn main() -> anyhow::Result<()> {
         process::exit(1);
     }
 
+    if args[1] == "--fixup-lockfile" {
+        if let Some(fixed) = fixup_lockfile(serde_json::from_str(&args[2])?)? {
```

Comment:
This is what I get for refactoring without testing 😅

```suggestion
        if let Some(fixed) = fixup_lockfile(serde_json::from_str(&fs::read_to_string(&args[2])?)?)? {
```

Will push when I can.

@reckenrode Do you mind making this change locally and testing?

Reply:
@winterqt It can confirm it builds now using the `github:` scheme. The only “issue” left is it needs me to set `makeCacheWritable` to `true` or it will fail when trying to write to `_cacache/tmp`.

Reply:
That's expected, due to how npm processes Git dependencies.

Reply:
That’s what I expected (hence the quotes). I was going to ask if it’s possible to change the location of the temporary directory, but it appears that it’s hardcoded as a child of `_cacache`.

Anyway, thanks for your work on this. Everything else seems to be working good now with my nix-foundryvtt derivation.